tests/contract/test_resource_client.py

```diff
@@ -476,6 +476,23 @@ def test_call_async(resource_client, action):
     assert response == {"status": OperationStatus.SUCCESS.value}
 
 
+@pytest.mark.parametrize("action", [Action.CREATE, Action.UPDATE, Action.DELETE])
```

Comment:
Is it worth adding a test for In_Progress with writeOnlyProperties in the response?

Reply:
Good idea, will do.